packages/tooltip/src/Tooltip.story.js

```diff
@@ -199,4 +199,36 @@ storiesOf("Tooltip/Tooltip", module)
         />
       </div>
     </div>
+  ))
+  .add("auto positioned considering TOP boundaries", () => (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        height: "350px",
+        background: "aliceblue",
+      }}
+    >
+      <div style={{ background: "beige", height: 150 }} />
```

Comment:
Need to increase this height. The tooltip is always showing at the bottom:
![2019-09-18 09 14 47](https://user-images.githubusercontent.com/1459283/65147626-ee9c9e80-d9f4-11e9-908f-7a1239cfdec5.gif)

Reply:
Fixed, thank